### ISSUE TITLE
Install netdata from EPEL repository for CentOS 7

### ISSUE THREAD

**Issue by Nirjonadda** [netdata/netdata#8784]
We have installed netdata from Linux manually Installation guide.

Enabled the EPEL repo
Enabled the additional repo from okay.network

And installed the minimum required dependencies

`yum install autoconf automake curl gcc git libmnl-devel libuuid-devel openssl-devel libuv-devel lz4-devel Judy-devel make nc pkgconfig python zlib-devel cmake libwebsockets-devel`

Then Installed Netdata.

`cd /root/netdata`

`./netdata-installer.sh`

But my question is how can update or install netdata from EPEL repository for CentOS 7?

`yum install netdata`

Do you need uninstall netdata from previous installation or can be install without uninstall?

**Comment by Ferroin** [comment]
You will need to uninstall the copy that you installed using `./netdata-installer.sh` if you want to use the EPEL package.

**Comment by Nirjonadda** [comment]
One more question, what the different between RPM `./netdata-installer.sh` vs YUM EPEL package?

**Comment by Nirjonadda** [comment]
OK we have installed EPEL package but `http://Server IP:19999/ `are not loading.

Unable to connect

Firefox can’t establish a connection to the server at IP:19999.

```
[root@na ~]# systemctl status netdata
● netdata.service - Real time performance monitoring
   Loaded: loaded (/usr/lib/systemd/system/netdata.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2020-04-22 01:05:48 +06; 6s ago
  Process: 15060 ExecStartPre=/bin/chown -R netdata:netdata /var/run/netdata (code=exited, status=0/SUCCESS)
  Process: 15057 ExecStartPre=/bin/mkdir -p /var/run/netdata (code=exited, status=0/SUCCESS)
  Process: 15054 ExecStartPre=/bin/chown -R netdata:netdata /var/cache/netdata (code=exited, status=0/SUCCESS)
  Process: 15052 ExecStartPre=/bin/mkdir -p /var/cache/netdata (code=exited, status=0/SUCCESS)
 Main PID: 15063 (netdata)
   CGroup: /system.slice/netdata.service
           ├─15063 /usr/sbin/netdata -P /var/run/netdata/netdata.pid -D
           ├─15156 /usr/bin/bash /usr/libexec/netdata/plugins.d/tc-qos-helper...
           ├─15161 /usr/libexec/netdata/plugins.d/apps.plugin 1
           ├─15165 /usr/bin/node /usr/libexec/netdata/plugins.d/node.d.plugin...
           └─15171 /usr/bin/python /usr/libexec/netdata/plugins.d/python.d.pl...

Apr 22 01:05:48 na.mysite.com systemd[1]: Starting Real time performance....
Apr 22 01:05:48 na.mysite.com systemd[1]: Started Real time performance ....
Apr 22 01:05:48 na.mysite.com netdata[15063]: SIGNAL: Not enabling reaper
Apr 22 01:05:48 na.mysite.com netdata[15063]: 2020-04-22 01:05:48: netdat...
Hint: Some lines were ellipsized, use -l to show in full.
[root@na ~]#
```

**Comment by prologic** [comment]
> OK we have installed EPEL package but `http://Server IP:19999/ `are not loading.
> 
> Unable to connect
> 
> Firefox can’t establish a connection to the server at IP:19999.

What is the output of `ss -tapn | grep netdata` or `netstat -tapn | netstats`?

**Comment by Nirjonadda** [comment]
@prologic We have installed EPEL package with `yum install netdata` but do we need more steps for netdata start for EPEL package?

```
[root@na ~]# ss -tapn | grep netdata
LISTEN     0      128    127.0.0.1:8125                     *:*                   users:(("netdata",pid=15063,fd=21))
LISTEN     0      128    127.0.0.1:19999                    *:*                   users:(("netdata",pid=15063,fd=5))
LISTEN     0      128      [::1]:8125                  [::]:*                   users:(("netdata",pid=15063,fd=20))
LISTEN     0      128      [::1]:19999                 [::]:*                   users:(("netdata",pid=15063,fd=4))
[root@na ~]# netstat -tapn | netstats
-bash: netstats: command not found
[root@na ~]#
```

**Comment by prologic** [comment]
It's right there:

```
LISTEN     0      128    127.0.0.1:19999                    *:*                   users:(("netdata",pid=15063,fd=5))
```

Reconfigure `/etc/netdata.conf` under the `[web]` section and change the `bind to =`.

**Comment by Nirjonadda** [comment]
This not working with change to `bind to = .` from `bind to = localhost`

**Comment by Nirjonadda** [comment]
OK this working with `bind to = Server IP`, So this change are not overwritten with updates? Also EPEL are not using latest version of netdata? How to update Latest version: v1.21.1 from EPEL repo?

![ScreenShot00298](https://user-images.githubusercontent.com/6899856/80093666-32481f80-8565-11ea-8a53-8c6f21be797a.png)

**Comment by prologic** [comment]
I'm really not sure wqhy you keep talking about NetData from EPEL repo(s). We do not push/maintain NetData in the EPEL repo. We _do_ however maintain RPM packages over at [PackageCloud](http://packagecloud.io/netdata/netdata) (_stable release_) and [Nightly Release](http://packagecloud.io/netdata/netdata-edge). Please refer to those.

**Comment by Ferroin** [comment]
@prologic There is, however, a Netdata package in EPEL (with things split out rather sensibly into multiple packages I might add).  You're right though that we don't maintain that.

**Comment by Nirjonadda** [comment]
@prologic Sorry, I have not seen that you have own RPM packages repo. So going to install PackageCloud repo. Thanks

**Comment by Nirjonadda** [comment]
Sorry, Can not install netdata from PackageCloud. We have uninstalled netadata from EPEL repo then try to install from PackageCloud repo.

```
[root@na ~]# yum install netdata
Loaded plugins: fastestmirror, universal-hooks
Loading mirror speeds from cached hostfile
 * EA4: 91.197.228.252
 * EA4-experimental: 91.197.228.252
 * cpanel-addons-production-feed: 91.197.228.252
 * cpanel-plugins: 91.197.228.252
 * base: mirror.seedvps.com
 * epel: epel.mirror.wearetriple.com
 * extras: mirror.prolocation.net
 * remi-safe: mirrors.ukfast.co.uk
 * updates: mirrors.xtom.nl
Resolving Dependencies
--> Running transaction check
---> Package netdata.x86_64 0:1.21.1-1.el7 will be installed
--> Processing Dependency: protobuf-c for package: netdata-1.21.1-1.el7.x86_64
--> Processing Dependency: protobuf for package: netdata-1.21.1-1.el7.x86_64
--> Running transaction check
---> Package protobuf.x86_64 0:2.5.0-8.el7 will be installed
---> Package protobuf-c.x86_64 0:1.0.2-3.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

=============================================================================================================================================================================================================================================
 Package                                                 Arch                                                Version                                                      Repository                                                    Size
=============================================================================================================================================================================================================================================
Installing:
 netdata                                                 x86_64                                              1.21.1-1.el7                                                 netdata_netdata                                              7.1 M
Installing for dependencies:
 protobuf                                                x86_64                                              2.5.0-8.el7                                                  base                                                         338 k
 protobuf-c                                              x86_64                                              1.0.2-3.el7                                                  base                                                          28 k

Transaction Summary
=============================================================================================================================================================================================================================================
Install  1 Package (+2 Dependent packages)

Total size: 7.4 M
Installed size: 31 M
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test


Transaction check error:
  file /usr/share/netdata/web/lib from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/console.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/bootstrap-3.3.7.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/bootstrap-slate-flat-3.3.7.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/bootstrap-slider-10.0.0.min.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/bootstrap-theme-3.3.7.min.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/bootstrap-toggle-2.2.2.min.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/c3-0.4.18.min.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/css/morris-0.5.1.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dash-example.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard.slate.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard_info.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/dashboard_info_custom_example.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/demo.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/demo2.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/demosites.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/demosites2.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/favicon.ico from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/favicon.ico from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts/glyphicons-halflings-regular.eot from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts/glyphicons-halflings-regular.svg from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts/glyphicons-halflings-regular.ttf from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts/glyphicons-halflings-regular.woff from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/fonts/glyphicons-halflings-regular.woff2 from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/goto-host-from-alarm.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/alert-128-orange.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/alert-128-red.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/alert-multi-size-orange.ico from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/alert-multi-size-red.ico from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-144x144.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-192x192.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-36x36.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-48x48.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-72x72.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/android-icon-96x96.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/animated.gif from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-114x114.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-120x120.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-144x144.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-152x152.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-180x180.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-57x57.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-60x60.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-72x72.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-76x76.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon-precomposed.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/apple-icon.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/banner-icon-144x144.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/check-mark-2-128-green.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/check-mark-2-multi-size-green.ico from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/favicon-16x16.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/favicon-32x32.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/favicon-96x96.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/ms-icon-144x144.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/ms-icon-150x150.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/ms-icon-310x310.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/ms-icon-70x70.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/netdata-logomark.svg from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/static/img/netdata-logomark.svg from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/netdata.svg from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/images/post.png from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/index.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/infographic.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/bootstrap-3.3.7.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/bootstrap-slider-10.0.0.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/bootstrap-table-1.11.0.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/bootstrap-table-export-1.11.0.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/bootstrap-toggle-2.2.2.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/clipboard-polyfill-be05dad.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/d3-4.12.2.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/d3pie-0.2.1-netdata-3.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/dygraph-c91c859.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/dygraph-smooth-plotter-c91c859.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/fontawesome-all-5.0.1.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/gauge-1.3.2.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/jquery-2.2.4.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/jquery.easypiechart-97b5824.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/jquery.peity-3.2.0.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/jquery.sparkline-2.1.2.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/lz-string-1.4.4.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/pako-1.0.6.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/perfect-scrollbar-0.6.15.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/lib/tableExport-1.6.0.min.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/main.css from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/main.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/netdata-swagger.json from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/netdata-swagger.yaml from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/refresh-badges.js from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/robots.txt from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/sitemap.xml from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/static from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/static/img from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/tv.html from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /usr/share/netdata/web/version.txt from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-data-1.20.0-1.el7.noarch
  file /etc/netdata/netdata.conf from install of netdata-1.21.1-1.el7.x86_64 conflicts with file from package netdata-conf-1.20.0-1.el7.noarch

Error Summary
-------------

[root@na ~]#
```

**Comment by Ferroin** [comment]
You didn't completely remove all of the packages from the EPEL packaging of Netdata. Either `yum autoremove` or explicitly uninstall `netdata-data` and `netdata-conf`.

**Comment by Nirjonadda** [comment]
@Ferroin 

Does this correct step completely remove all of the packages from the EPEL packaging of Netdata?

```
yum remove netdata
yum remove netdata-data
yum remove netdata-conf
```

**Comment by Ferroin** [comment]
@Nirjonadda I believe so.

**Comment by prologic** [comment]
> @prologic There is, however, a Netdata package in EPEL (with things split out rather sensibly into multiple packages I might add). You're right though that we don't maintain that.

Oh dear I didn't know that :/ This sort of collides with our own packaging efforts a bit doesn't it?

**Comment by prologic** [comment]
@Nirjonadda I _assume_ by the issue closure you managed to successfully instlal NetData with our RPM packages from our PackageCloud repo finally?

**Comment by Ferroin** [comment]
> > @prologic There is, however, a Netdata package in EPEL (with things split out rather sensibly into multiple packages I might add). You're right though that we don't maintain that.
> 
> Oh dear I didn't know that :/ This sort of collides with our own packaging efforts a bit doesn't it?

Many distros package Netdata. See https://repology.org/project/netdata/versions for a rundown of the major ones. EPEL, however, is one of the only ones that splits things out like this (but I would argue we should at least split out the dashboard like they do).

**Comment by zystem** [comment]
This change brokes installation from official (https://packagecloud.io/netdata/netdata/el/7/$basearch) repo
`[root@jenkins netdata]# yum install --enablerepo=netdata_netdata netdata netdata-conf netdata-data
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos.interhost.net.il
 * extras: centos.interhost.net.il
 * updates: centos.interhost.net.il
Package netdata-conf is obsoleted by netdata, trying to install netdata-1.31.0-1.el7.x86_64 instead
Package netdata-data is obsoleted by netdata, trying to install netdata-1.31.0-1.el7.x86_64 instead
Resolving Dependencies
--> Running transaction check
---> Package netdata.x86_64 0:1.31.0-1.el7 will be installed
--> Processing Dependency: netdata-conf = 1.31.0-1.el7 for package: netdata-1.31.0-1.el7.x86_64
Package netdata-conf is obsoleted by netdata, but obsoleting package does not provide for requirements
--> Processing Dependency: netdata-data = 1.31.0-1.el7 for package: netdata-1.31.0-1.el7.x86_64
Package netdata-data is obsoleted by netdata, but obsoleting package does not provide for requirements
--> Finished Dependency Resolution
Error: Package: netdata-1.31.0-1.el7.x86_64 (epel)
           Requires: netdata-conf = 1.31.0-1.el7
           Available: netdata-conf-1.31.0-1.el7.noarch (epel)
               netdata-conf = 1.31.0-1.el7
Error: Package: netdata-1.31.0-1.el7.x86_64 (epel)
           Requires: netdata-data = 1.31.0-1.el7
           Available: netdata-data-1.31.0-1.el7.noarch (epel)
               netdata-data = 1.31.0-1.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
`
